frontend/packages/component-library/src/section/Section.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import {HTMLAttributes, ReactNode} from 'react';
+
+import moduleStyles from './section.module.scss';
+
+export interface SectionProps extends HTMLAttributes<HTMLElement> {
+  /** Background color */
+  backgroundColor?:
+    | 'primary'
+    | 'secondary'
+    | 'dark'
+    | 'brand-light-primary'
+    | 'brand-light-secondary';
+  /** Background image */
+  backgroundImage?: string;
+  /** Background image repeat */
+  backgroundImageRepeat?: boolean;
+  /** Background size */
+  backgroundSize?: 'contain' | 'cover';
+  /** Vertical padding */
+  padding?: 'm' | 'l';
```

Comment:
@levadadenys I didn't use `ComponentSizeXSToL` here since we're only using two sizes, but let me know if there's a way to only use `m` and `l` from there and I'll swap it out.